source/scripts/app.js

```diff
@@ -808,7 +810,7 @@ async function populateSelectCookbookOptions() {
 
   let cookbooks = await indexedDb.getAllCookbooks();
 
-  for (let i = 0; i < cookbooks.length; ++i) {
+  for (let i = 1; i < cookbooks.length; ++i) {
     let option = document.createElement("option");
     option.value = cookbooks[i].title;
     option.textContent = cookbooks[i].title;
```

Comment:
This implementation has a chance of populating the default cookbook twice, do this instead
```
for (let i = 0; i < cookbooks.length; ++i) {
  if (cookbooks[i].title !== DEFAULT_COOKBOOK_NAME) {
    let option = document.createElement("option");
    option.value = cookbooks[i].title;
    option.textContent = cookbooks[i].title;
    cookbookDropdown.append(option);
  }
}
```